pom.xml

```diff
@@ -2981,6 +2981,9 @@
                     <maxJdkVersion>${java.version}</maxJdkVersion>
                     <ignoredScopes>test</ignoredScopes>
                     <ignoredScopes>provided</ignoredScopes>
+                    <ignoreClasses>
+                      <ignoreClass>org.jline.terminal.impl.ffm.*</ignoreClass>
```

Comment:
Please add a comment with the pointer https://github.com/scala/bug/issues/12994 .